sdk/src/main/scala/com/horizen/account/block/AccountBlock.scala

```diff
@@ -35,6 +37,8 @@ class AccountBlock(override val header: AccountBlockHeader,
 
   override lazy val transactions: Seq[SidechainTypes#SCAT] = sidechainTransactions
 
+  def ethereumTransactions: Seq[EthereumTransaction] = sidechainTransactions.map(_.asInstanceOf[EthereumTransaction])
```

Comment:
Move to a util like Alex suggested, or maybe make this non-specific to Ethereum:
```suggestion
  def transactionsAs[T]: Seq[T] = sidechainTransactions.map(_.asInstanceOf[T])
```